server/sandbox_run_linux.go

```diff
@@ -482,6 +484,23 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	if err != nil {
 		return nil, err
 	}
+
+	runtimeType, err := s.Runtime().ContainerRuntimeType(container)
+	if err != nil {
+		return nil, err
+	}
+	// If using kata runtime, the process label should be set to container_kvm_t
+	// Mind that kata does *not* apply any process label to the VM workloads.
```

Comment:
suggested rewording:
`Keep in mind that kata does *not* apply any process label to containers within the VM`

Reply:
Suggestion taken!